packages/oscal-react-library/src/components/OSCALControl.js

```diff
@@ -44,6 +45,15 @@ function ControlsList(props) {
   } = props;
   return (
     <div>
+      <OSCALPropertiesDialog
+        properties={control.props}
+        title={
+          <>
+            <OSCALControlLabel id={control.id} label={control.label} component="span" />
+            {` ${control.title}`}
+          </>
+        }
+      />
```

Comment:
Is there a cleaner way to deal with this? Is there a reason each component using an `OSCALPropertiesDialog` would need to create there own title? Would having this just be the `children` be a cleaner solution?

Reply:
I personally would like if we stopped putting huge components as a prop. At the very least lets do something like

```TypeScript
const controlLabel =  (
  <>
    <OSCALControlLabel id={control.id} label={control.label} component="span" />
      {` ${control.title}`}
  </>
);

return (
  <OSCALPropertiesDialog title={controlLabel />
);